qpsolvers_benchmark/benchmark.py

```diff
@@ -175,43 +183,63 @@ def parse_command_line_arguments():
 
 
 def find_results_file(args):
```

Comment:
```suggestion
def find_results_file(args: argparse.Namespace) -> str:
```
Type annotations are understood by Sphinx for the docs, with the extra benefit that they will be checked by mypy in the CI.